app/controllers/medicaid/intro_location_help_controller.rb

```diff
@@ -2,5 +2,16 @@
 
 module Medicaid
   class IntroLocationHelpController < StandardStepsController
+    include MedicaidFlow
+
+    private
+
+    def skip?
```

Comment:
this will be tested via feature spec when we add one -- when should we do that, @jayroh ?

Reply:
If we have unit tests for the controllers we _could_ delay the feature spec until we have a little more meat in the medicaid side. I think it depends on what we would consider the happy path?

OR we can just start now? 😄